src/views/communities/communities.module.css

```
@@ -177,6 +177,18 @@
 }
 
+.directoryMarker {
+  font-size: x-small;
+  color: var(--text);
+}
+
+.directoryServedBy {
+  font-size: x-small;
+  color: var(--text);
+  text-transform: lowercase;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)